Core/clim-core/presentations/presentation-types.lisp

```diff
@@ -57,6 +57,32 @@
   (:documentation "The specializer to use for this type in a presentation
 method lambda list"))
 
+(defmethod ptype-specializer ((type symbol))
+  (if-let ((ptype (or (find-presentation-type type nil)
+                      (find-class type nil))))
+    (ptype-specializer ptype)
+    ;; Assume it's a forward referenced CLOS class.
+    type))
+
+(defmethod ptype-specializer ((type standard-class))
+  (class-name type))
+
+(defmethod ptype-specializer ((type cons))
+  (if (typep type '(cons (eql eql)
+                    (cons t null)))
+      type
+      (error "~s is not a valid presentation method specializer." type)))
```

Comment:
Could you add a simple test for these?

Reply:
I did, see later commits, most notably `Tests/presentations/presentation-functions.lisp`,eql specializer is used with the method ulala. unless you mean, that we should unit test the internal utility `ptype-specializer` -- I think that testing method type specializers directly is better.

Reply:
I commented while reading, so I didn't know at that point, sorry.